interface/src/framework/system/GeneralFileUpload.tsx

```diff
@@ -11,16 +23,94 @@ interface UploadFileProps {
 const GeneralFileUpload: FC<UploadFileProps> = ({ uploadGeneralFile }) => {
   const [uploadFile, cancelUpload, uploading, uploadProgress] = useFileUpload({ upload: uploadGeneralFile });
 
+  const { me } = useContext(AuthenticatedContext);
+  const { enqueueSnackbar } = useSnackbar();
+
+  const saveFile = (json: any, endpoint: string) => {
+    const a = document.createElement('a');
+    const filename = 'emsesp_' + endpoint + '_json.txt';
```

Comment:
This should stay as *.json, because:
- the UploadFileService.cpp checks for *.json extension
- it prevents users drop this file to GH issue.

Reply:
forgot about that check. I thought you wanted the files to be .txt so they can attached to a GH issue, as mentioned in https://github.com/emsesp/EMS-ESP32/issues/573#issuecomment-1186852058 ?

Reply:
oh never mind, I see its only from the Help. I'll fix it